README.md

```
@@ -42,6 +42,8 @@ Are you missing an example? Please feel free to [open an issue](https://github.c
 
 | Demo                                                                                                 | Description                                                                                                                                                                                |
 | ---------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [`nextjs`](https://github.com/prisma/prisma-examples/tree/latest/orm/nextjs)   | [Next.js](https://nextjs.org/) app with a REST API (using [Next.js API routes](https://nextjs.org/docs/api-routes/introduction))                                                           |
+[`nextjs-auth`](https://github.com/prisma/prisma-examples/tree/latest/orm/nextjs)   | [Next.js](https://nextjs.org/) app with Auth.js(https://authjs.dev/) (using [Auth.js]([https://nextjs.org/docs/api-routes/introduction](https://authjs.dev/reference/nextjs))) |
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)